docs/adr/0000_architecture_decisions_records.md

```diff
@@ -0,0 +1,60 @@
+---
+content-version: 0.1.77
+title: ADR-0000 - Architecture Decisions Records
+status: proposed
```

Comment:
Do we want to merge with this set to "accepted" vs "proposed"?

Reply:
Done. I think it is already in a good state after reviews and feedback.